tests/benchmarks/constructs/OperationIntegerJinjaAdd.py

```diff
@@ -0,0 +1,17 @@
+{% extends "layout" %}
```

Comment:
I think "layout" is a bad name, can you come up with some more descriptive. How about ``RepeatedFunctionCall.py.j2`` or so. And use the `.j2` suffix, so we can tell Jinja templates, even if there is no official suffix for these kinds of files, but this way, ``autoformat-nuitka-source`` could learn some basic corrections.